src/app/core/views/Preferences/tabs/Security/BackupKey.tsx

```diff
@@ -8,33 +8,34 @@ import Section from '../../components/Section';
 import { TrackingPlan } from 'app/analytics/TrackingPlan';
 import { trackBackupKeyDownloaded } from 'app/analytics/services/analytics.service';
 
-export default function BackupKey({ className = '' }: { className?: string }): JSX.Element {
-  const { translate } = useTranslationContext();
+export function handleExport(translate) {
```

Comment:
Add jsdoc to this function, and it might be a good idea to split it into several functions.